### ISSUE TITLE
Compare 2 branches history

### ISSUE THREAD

**Issue by wctrl** [extrawurst/gitui#2272]
In lazygit I can compare 2 branches (remote and local, on screenshot):
![2024-06-18-164933_hyprshot](https://github.com/extrawurst/gitui/assets/166127285/3ebba61c-efb1-4af4-b80e-566356148867)

Does gitui have same thing?

**Comment by extrawurst** [comment]
yeah, you scroll to any branch local or remote in the branch popup and compare against the current one you are on

**Comment by wctrl** [comment]
I see there only compare only files option, can gui compare **the history**?
